tests/test_rules_mapper_bibs.py

```diff
@@ -316,7 +316,6 @@ def test_should_add_identifiers_010_019_020_022_024_028_035(mapper):
     ]
     expected_identifiers = [
         "(OCoLC)ocn898162644",
-        "19049386",
```

Comment:
Were the tests failing here for you? @jermnelson

Reply:
@bltravis this test was failing for me because the value `19049386` is coming from the [001 field ](https://github.com/FOLIO-FSE/folio_migration_tools/blob/7bbcf2f586529bc483eadb55c6bee900ed2b191d/tests/test_data/default/test_identifiers.xml#L15)and the rules in the default_mapper didn't include the 001. I wasn't sure if this value should be included in the list of expected identifiers or if  we need to add a new rule for `001`.

Reply:
@jermnelson 🤔 It doesn't fail with the pre-5.0.0 version of pymarc, so something is going on there.

Reply:
@bltravis this is now fixed with the test restored with an update to the hrid_handler.handle_035_generation in this [commit](https://github.com/FOLIO-FSE/folio_migration_tools/pull/643/commits/aec0454e379d26fcddaf46a4d66600d8cf5a35e4).